README.md

```diff
@@ -1,70 +1,65 @@
-# Getting Started with Create React App
+# React and Redux
```

Comment:
- You currently still have the default microverse README template. Please make sure you have a professional README file.